_config.yml

```diff
@@ -18,8 +18,8 @@ defaults:
     scope:
       path: ""
     values:
-      fullversion: "v1.7.0"
-      version: "v1.7"
+      fullversion: "v1.8.0"
+      version: "v1.8"
       githubbranch: "master"
       docsbranch: "master"
       versions:
```

Comment:
You'll probably want to add a section in here for 1.8. I should also create a subdomain for 1.7.